static/index.html

```diff
@@ -34,6 +34,36 @@
 			}
 		}
 	}
+	async function fmtCode() {
+		const element = document.getElementById('code');
+		const stdout = document.getElementById('stdout');
+		if (element && stdout) {
+			const code = element.value;
+			try {
+				const res = await fetch('/server/fmt/', {
+					method: 'POST',
+					headers: {
+						'Content-Type': 'text/plain'
+					},
+					body: code
+				});
+				const text = await res.text();
+				let msg = text;
+				if (res.status === 429) {
+					msg = 'Too many requests. Please wait a minute and then try again.';
+				} else if (!res.ok) {
+					msg = 'An error occurred:\n' + text;
+				} else if (res.ok) {	
+					msg = 'zig fmt successfull\n';
```

Comment:
Typo, should be _successful_.

Reply:
https://github.com/gsquire/zig-play/pull/11/commits/164fa2e87b0c9ecfb11d1603eded02e32d5483b5